components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -38,7 +38,8 @@
  in a sheet from the bottom.
 
  */
-__attribute__((objc_subclassing_restricted)) @interface MDCActionSheetController : UIViewController
+__attribute__((objc_subclassing_restricted)) @interface MDCActionSheetController
```

Comment:
This is protocol is available on iOS 10+. Are we planning to keep the MDC APIs deprecated until we drop iOS 9 supporrt?